common/lib/api-client/index.js

```diff
@@ -22,7 +24,7 @@ module.exports = function () {
 
   instance = new JsonApi({
     apiUrl: API.BASE_URL,
-    logger: IS_DEV,
+    logger: false,
```

Comment:
Any reason for turning this off now, even for development?

Reply:
See https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/801#discussion_r492631417

Reply:
Offtopic, so don't want to derail this PR - but `devour` using its own internal logger will get in the way of any json logging approach we'll want to do in the future.

Reply:
exactly